marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteDataTest.java

```diff
@@ -132,6 +132,20 @@ void invalidPermissionsString() {
         assertTrue(ex.getCause().getMessage().contains("Unable to parse permissions: rest-reader,read,rest-writer"), "Unexpected error message: " + ex.getCause().getMessage());
     }
 
+    @Test
+    public void testEndpointsAreLoaded() throws Exception {
+        TextDocumentManager modMgr = getHubClient().getModulesClient().newTextDocumentManager();
+        modMgr.delete("/data-hub/5/data-services/ingestion/bulkIngester.api", "/data-hub/5/data-services/ingestion/bulkIngester.sjs",
```

Comment:
This PR should including deleting the two files from /data-hub/5/data-services/ingestion from git, as they're being moved to /marklogic-data-hub-spark-connector/

Reply:
Should we keep these in datahub modules too? Just in case if installer is installing these modules?

Reply:
The modules should still be in DHF core ,but under /marklogic-data-hub-spark-connector/